tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -79,6 +79,68 @@ public function testFixCollapse()
      * @param  mixed    &$reference     A parameter passed by reference
      */
 
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testFixPartiallyUntyped()
+    {
+        $this->fixer->configure(['tags' => ['param']]);
+
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @param         $id
+     * @param         $parentId
+     * @param int     $websiteId
+     * @param         $position
+     * @param int[][] $siblings
+     */
+
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    /**
+     * @param      $id
+     * @param    $parentId
+     * @param int $websiteId
+     * @param        $position
+     * @param int[][]  $siblings
+     */
+
+EOF;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testFixPartiallyUntypedCollapse()
+    {
+        $this->fixer->configure(['tags' => ['param'], 'collapse' => true]);
+
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @param $id
```

Comment:
fixed only bug I found locally for my new flag: partially untyped annotations came with extra spaces